must-gather/gather

```diff
@@ -9,11 +9,12 @@ for I in $(/usr/bin/oc get crd | grep local.storage.openshift.io | awk '{print $
 done
 
 lsoNamespace=$(/usr/bin/oc get subscription --all-namespaces | grep ${subscriptionName} | awk '{print $1}')
-for I in $(/usr/bin/oc get pod -n ${lsoNamespace} | tail -n +2 | awk '{print $1}'); do
+#for I in $(/usr/bin/oc get pod -n ${lsoNamespace} | tail -n +2 | awk '{print $1}'); do
```

Comment:
can you please remove this line if the PR works?

Reply:
Done.